EIPS/eip-4200.md

```diff
@@ -47,11 +47,11 @@ If the code is valid EOF1:
 
 1. `RJUMP relative_offset` sets the `PC` to `PC_post_instruction + relative_offset`.
 2. `RJUMPI relative_offset` pops a value (`condition`) from the stack, and sets the `PC` to `PC_post_instruction + ((condition == 0) ? 0 : relative_offset)`.
-3. `RJUMPV count relative_offset+` pops a value (`case`) from the stack, and sets the `PC` to `PC_post_instruction + ((case >= (count + 1)) ? 0 : relative_offset[case])`.
+3. `RJUMPV count relative_offset+` pops a value (`case`) from the stack, and sets the `PC` to `PC_post_instruction + ((case >= count) ? 0 : relative_offset[case])`.
```

Comment:
```suggestion
3. `RJUMPV count relative_offset*` pops a value (`case`) from the stack, and sets the `PC` to `PC_post_instruction + ((case >= count) ? 0 : relative_offset[case])`.
```

Reply:
but empty table is not allowed, so there's always at least 1 offset

Reply:
Yes, but that is not a rule of instruction encoding, but validation :)

Reply:
I think `*` would be confusing, merging with `+`